### DISCUSSION TITLE
Roadmap

### DISCUSSION THREAD

**Discussion by apanicker-nflx** [Netflix/conductor#2254] [Announcements]
The [roadmap](https://github.com/Netflix/conductor/wiki/Roadmap) for Conductor development has been published. This is not an exhaustive list of features and enhancements. If there are features/enhancements that you think are worth adding to Conductor and will be beneficial to the community, please start a new discussion to gauge interest and gather feedback from the community.

We welcome contributions from the community. Please refer the [contributing guide](https://github.com/Netflix/conductor/blob/main/CONTRIBUTING.md) for guidelines.

**Comment by venkag** [comment]
@apanicker-nflx 
Just curious, is there a timeline you are targeting for this roadmap?

**Reply by aravindanr** [reply]
There is no specific timeline for it. Items in each sub-section are ordered by priority. Our hope is that we can address the top 2 items in each sub-section within a year. If any of those items piqued your interest, feel free to start a discussion.

**Reply by venkag** [reply]
Thanks @aravindanr.
